masterlist.yaml

```
@@ -7722,6 +7722,75 @@ plugins:
         crc: 0x68A235DA
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 6
+       
+  - name: 'AncientLand2.esp'
```

There's some excess whitespace here:

```suggestion
        itm: 6

  - name: 'AncientLand2.esp'
```